solidity/contracts/ViewSPV.sol

```diff
@@ -96,11 +100,10 @@ library ViewSPV {
     /// @param _headers     Raw byte array of header chain
     /// @return             The total accumulated difficulty of the header chain, or an error code
     function checkChain(bytes29 _headers) internal view returns (uint256 _totalDifficulty) {
-        bytes29 _headerChain = _headers.tryAsHeaderArray().assertValid();
```

Comment:
any reason to remove without replacing? I guess `indexHeaderArray` will make this check later :thinking:

Reply:
Oh, thanks!  I added the validation check in `ViewSPVtest.sol`, but forgot to add the type assertion here.